### PR TITLE
Add first achievements

### PR DESCRIPTION
**Pull request by conchyliculture**
We show a specific message for first beer ever, and every large amount of litres

### CODE REVIEW

File: beerlog/beerlogdb.py [hunk 1]
```diff
@@ -125,6 +125,22 @@ def GetScoreBoard(self):
...
+  def GetGlassFromName(self, name):
+    """Returns the corresponding glass from a uid
+
+    Args:
+      name(str): the character name.
+    Returns:
+      int: the glass size for a character, or the default value if not found.
+    """
+
+    entry = Entry.select(Entry.amount).where(
```
Comment (pilou-) [inline beerlog/beerlogdb.py:137]: Not sure if using `get` would change something.

File: beerlog/gui/display_tests.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        events_queue=events_queue, database=self.db)
+
+    self.db.AddEntry('0x0', 'pic')
+    achievements = d.GetAchievements('toto')
+    self.assertEqual(achievements[0].message, 'First beer, enjoy the run!')
+
+    # Go just over 5L
+    for _ in range(14):
+      self.db.AddEntry('0x0', 'pic')
+    events_queue = multiprocessing.Queue()
```
Comment (pilou-) [inline beerlog/gui/display_tests.py:44]: This line seems extraneous.

File: beerlog/beerlogdb.py (hunk 1 above)
Comment (conchyliculture) [reply beerlog/beerlogdb.py:137]: sure

File: beerlog/gui/display_tests.py (hunk 2 above)
Comment (conchyliculture) [reply beerlog/gui/display_tests.py:44]: fixed

**Pull request merged**